pandas/core/indexers/objects.py

```diff
@@ -215,12 +224,14 @@ def get_window_bounds(
         min_periods: int | None = None,
         center: bool | None = None,
         closed: str | None = None,
+        step: int | None = None,
     ) -> tuple[np.ndarray, np.ndarray]:
 
-        return (
-            np.zeros(num_values, dtype=np.int64),
-            np.arange(1, num_values + 1, dtype=np.int64),
-        )
+        if step is None:
```

Comment:
So generally in this file I think `get_window_bounds` should accept a `step` parameter but the rest of the logic should be unchanged (except for `FixedWindowIndexer.get_window_bounds` of course)